bert_pretrain/main.py

```diff
@@ -87,14 +87,23 @@ def build_model(cfg):
             tokenizer_name=cfg.get('tokenizer_name', None),
             gradient_checkpointing=cfg.get('gradient_checkpointing', None)
         )
-    elif cfg.name == 'mosaic_bert':
+    elif cfg.name == 'mosaic_bert_mlm':
         return create_mosaic_bert_mlm(
             pretrained_model_name=cfg.pretrained_model_name,
             use_pretrained=cfg.get('use_pretrained', None),
             model_config=cfg.get('model_config', None),
             tokenizer_name=cfg.get('tokenizer_name', None),
             gradient_checkpointing=cfg.get('gradient_checkpointing', None)
         )
+    elif cfg.name == 'mosaic_bert_classification':
```

Comment:
I'd opt for reverting these changes to `main.py` because this script is dedicated to MLM pre-training.
We'll actually only want to call `create_mosaic_bert_classification` in the fine-tuning script, which currently lives in a different branch, but we can address that later.

Reply:
That seems reasonable. I will revert these changes.